src/ui/lp-render.ts

```diff
@@ -330,11 +330,13 @@ export function inlinePlugin(app: App, index: FullIndex, settings: DataviewSetti
                         try {
                             // for setting the correct context for dv/dataview
                             const myEl = createDiv();
-                            const dvInlineApi = new DataviewInlineApi(api, this.component, myEl, currentFile.path);
+                            const dvInlineApi = new DataviewInlineApi(api, this.component, code.includes("await") ? el : myEl, currentFile.path);
                             if (code.includes("await")) {
```

Comment:
oshit this is nasty.
Not related to this PR as much, but this `code.includes("await")` is a shotgun.